data.h

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "cryptopp/sha.h"
+#include "cryptopp/Integer.h"
+
+class Data {
+  public:
+    const static unsigned int kBlockSize = 16;
+    const static unsigned int kDigestSize = 32;
+
+    // Empty constructor
+    Data();
+    // Converts byte array into Data object
+    Data(byte* data, size_t sizeIn = kBlockSize);
+    // Converts Integer into Data object
+    Data(IROHA_CRYPTO_HH::Integer data, unsigned int size = kBlockSize);
+    ~Data();
+
+    // Encodes byte array of object into base64 string
+    std::string toString();
+    // Returns the size of the data object
+    size_t size();
+    byte bytes[kDigestSize];
+
+    /*------ Static functions ------*/
+    // Hashes a message to size bytes: default 32 bytes
+    static Data hashMessage(std::string message, int messageLen, int size=kDigestSize);
```

Comment:
`message.size() == messageLen`? second arg seem redundant

Reply:
Thanks @l4l for pointing this.